[PyTorch] Try using ExclusivelyOwned in LinearAlgebra

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#59420**
* #59827

This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

swolchok committed:
[PyTorch] Try using ExclusivelyOwned in LinearAlgebra

This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

ezyang commented:
@mruberry in my glorious transpiler future, I would like it if the transpiler can automatically apply this transformation

mruberry commented:
> @mruberry in my glorious transpiler future, I would like it if the transpiler can automatically apply this transformation

Added a note to the Quip

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

ezyang reviewed (approved):
This is pretty non-invasive, so thumbs up for me, if you can get it to build, anyway.

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] Try using ExclusivelyOwned in LinearAlgebra"


This is a sample of how we might use ExclusivelyOwned on an opt-in basis.

Differential Revision: [D28885313](https://our.internmc.facebook.com/intern/diff/D28885313/)

[ghstack-poisoned]
